wpilib/wpilib/serialport.py

```diff
@@ -35,32 +36,32 @@ class SerialPort:
         http://www.ni.com/pdf/manuals/370132c.pdf
     """
     
-    class Port:
+    class Port(enum.IntEnum):
         kOnboard = 0
         kMXP = 1
         kUSB = 2
         kUSB1 = 2
         kUSB2 = 3
             
-    class Parity:
+    class Parity(enum.IntEnum):
         kNone = 0
         kOdd = 1
         kEven = 2
         kMark = 3
         kSpace = 4
             
-    class StopBits:
+    class StopBits(enum.IntEnum):
         kOne = 10
         kOnePointFive = 15
         kTwo = 20
             
-    class FlowControl:
+    class FlowControl(enum.IntFlag):
```

Comment:
In case you didn't realise already from the CI logs, `enum.Flag` is new in Python 3.6. Unless we drop support for Python 3.5, we can't use it.

Reply:
Sounds like....we should...drop support 👍

Reply:
You'll have to give some compelling reasons to drop support for Python 3.5 though.

- Debian stable (stretch) ships Python 3.5.
- Ubuntu 16.04 (the second-latest LTS release, which is still supported) ships with Python 3.5.